Since perl is used in just one place and can easily be replaced by a
little bit of shell, do so.  This makes testing on limited systems
easier.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 INSTALL.txt | 1 -
 test.sh     | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/INSTALL.txt b/INSTALL.txt
index 256beab..f77cbe6 100644
--- a/INSTALL.txt
+++ b/INSTALL.txt
@@ -67,7 +67,6 @@ In addition to the prerequisites mentioned above, you also need:

 To debug and run the performance test suite you'll also need:

-- Perl (http://www.perl.org/)
 - Python (http://www.python.org/)

 Run "./autogen.sh" and then follow the steps mentioned under "Installation"
diff --git a/test.sh b/test.sh
index f14e287..1090649 100755
--- a/test.sh
+++ b/test.sh
@@ -1834,7 +1834,7 @@ prepare_cleanup_test() {
     mkdir -p $dir
     i=0
     while [ $i -lt 10 ]; do
-        perl -e 'print "A" x 4017' >$dir/result$i-4017.o
+        printf '%4017s' '' | tr ' ' 'A' >$dir/result$i-4017.o
         touch $dir/result$i-4017.stderr
         touch $dir/result$i-4017.d
         if [ $i -gt 5 ]; then
--
